flexmemb.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+// chapter 14 exercise 12
+
+typedef struct flex
+{
+    size_t numberOfScores;
+    double averageScore;
+    double scores[];
+}Flex_t;
+
+static void printStructValue(const struct flex * flexStruct)
+{
+    printf("Scores : ");
+    for(int index = 0; index < flexStruct->numberOfScores; index++)
+    {
+        printf("%g ", flexStruct->scores[index]);
+    }
+    printf("\nAverage: %g\n", flexStruct->averageScore);
```

Comment:
Also do you want to format this so it shows a fixed number of characters (or none) beyond the decimal point?
```suggestion
    printf("\nAverage Score: %g\n", flexStruct->averageScore);
```